looker_deployer/commands/deploy_content.py

```diff
@@ -137,7 +142,7 @@ def deploy_space(s, sdk, env, ini, recursive, debug=False):
     logger.debug("files to process", extra={"looks": look_files, "dashboards": dash_files})
 
     # cut down directory to looker-specific paths
-    a, b, c = s.partition("Shared")  # Hard coded to Shared for now TODO: Change this!
+    a, b, c = s.partition(target_base)  
```

Comment:
Unsure what the exact issue is but regular export work fine, but imports do not (it just returns the CLI input for `usage: ldeploy`

The below 2 lines at a minimum should always run as just a basic integration test.

```
ldeploy content export --env gcpl224 --folders 1 --local-target ./gcpl224_new/

ldeploy content import --env gcpl224 --folders ./gcpl224/Shared --recursive
```

Reply:
Needed to add a default for target_base, line 301, otherwise would fail without a --target-folder